cmd/download_test.go

```diff
@@ -113,7 +113,7 @@ func TestDownload(t *testing.T) {
 		defer os.RemoveAll(tmpDir)
 		assert.NoError(t, err)
 
-		ts := fakeDownloadServer(tc.requestor)
+		ts := fakeDownloadServer(fmt.Sprintf("%v", tc.requester))
```

Comment:
`strconv.FormatBool(tc.requester)` is what you want here.

Reply:
Ooooh, love it. Thanks!